custom_metrics/css.js

```diff
@@ -0,0 +1,24 @@
+//[css]
+// Uncomment the previous line for testing on webpagetest.org
+
+return JSON.stringify({
+  'has_cssinjs_styled_components': !!(document.querySelector('style[data-styled]') || document.querySelector('style[data-styled-components]')),
+  'has_cssinjs_radium': !!document.querySelector('[data-radium]'),
+  'has_cssinjs_jss': !!document.querySelector('[data-jss]'),
+  'has_cssinjs_emotion': !!document.querySelector('[data-emotion]'),
+  'has_cssinjs_goober': !!document.getElementById('_goober'),
+  'has_cssinjs_merge-styles': !!document.querySelector('[data-merge-styles]'),
+  'has_cssinjs_jsx': (() => {
+    let hasJSX = false;
+    document.querySelectorAll('style').forEach((s) => {
+      hasJSX = hasJSX || s.id.indexOf('__jsx-') === 0
+    });
+
+    return hasJSX;
+  })(),
```

Comment:
Can you use an attribute selector for this?
```suggestion
  'has_cssinjs_jsx': !!document.querySelector('style[id*="__jsx-"]'),
```

Reply:
Thanks, done